### PR TITLE
Is connected

### PR DESCRIPTION
**Pull request by zoepage**

### CODE REVIEW

File: lib/is-connected.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+module.exports = function isConnected (state, options) {
+  console.log(!!state.replication)
```
Comment (gr2m) [inline lib/is-connected.js]: remove the console.log, please :)

File: tests/specs/is-connected.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  t.equal(api.isConnected(), false, 'connection is not open')
+})
+
+test('api.isConnected() after connected', function (t) {
+  t.plan(1)
+  var db2 = dbFactory('isConnectedDB3')
+  var api = db2.hoodieSync({remote: 'isConnectedDB4'})
+
+  api.connect().then(function () {
+    t.equal(api.isConnected(), true, 'connection was opened')
```
Comment (gr2m) [inline tests/specs/is-connected.js]:
```diff
- connection was opened
+ connection is opened
```

File: tests/specs/is-connected.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  })
+})
+
+test('api.isConnected() after disconnected', function (t) {
+  t.plan(1)
+  var db3 = dbFactory('isConnectedDB5')
+  var api = db3.hoodieSync({remote: 'isConnectedDB6'})
+
+  api.disconnect().then(function () {
+    t.equal(api.isConnected(), false, 'connection was closed')
```
Comment (gr2m) [inline tests/specs/is-connected.js]:
```diff
- connection is not open
- connection is closed again
```

**Comment by zoepage** [comment]
updated

File: tests/specs/is-connected.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+var test = require('tape')
+var dbFactory = require('../utils/db')
+
+/* create if db does not exist, ping if exists or created */
+test('api.isConnected()', function (t) {
+  t.plan(1)
+  var db1 = dbFactory('isConnectedDB1')
+  var api = db1.hoodieSync({remote: 'isConnectedDB2'})
+
+  t.equal(api.isConnected(), false, 'connection is closed again')
```
Comment (gr2m) [inline tests/specs/is-connected.js]: why "again"? I'd just say `connection is closed`

File: tests/specs/is-connected.js (hunk 3 above)
Comment (gr2m) [inline tests/specs/is-connected.js]: why did you change it to `connection was closed`? I'd say `connection is closed again` is still better here

**Comment by zoepage** [comment]
updated
